src/kata5.reachDestination.js

```diff
@@ -1,3 +1,9 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+  const roundUp = Math.ceil((distance / speed) * 2) / 2;
+
+  console.log(roundUp);
```

Comment:
Should always remove console logs once youre done with them! Running the tests the screen is full of them for no good reason, these should be removed throughout the whole project